### ISSUE TITLE
minGW build fails on travis-ci

### ISSUE THREAD

**Issue by docbacardi** [muhkuh-sys/wxlua#1]
Building the project fails on travis-ci during 'configure' for wxWidgets. It claims that 'windows.h' can not be found. The same build script works on my Ubuntu 12.04 VM.

checking for w32api.h... no
checking for windows.h... no
configure: error: please set CFLAGS to contain the location of windows.h

**Comment by docbacardi** [comment]
On Travis `gcc` is used instead of `i686-w64-mingw32-gcc`. Here is the difference in the configure output.
Works:

```
configure:16389: checking for i686-w64-mingw32-gcc
configure:16405: found /usr/bin/i686-w64-mingw32-gcc
configure:16416: result: i686-w64-mingw32-gcc
configure:16685: checking for C compiler version
configure:16694: i686-w64-mingw32-gcc --version >&5
i686-w64-mingw32-gcc (GCC) 4.8.2
```

Fails:

```
configure:16389: checking for i686-w64-mingw32-gcc
configure:16416: result: gcc
configure:16685: checking for C compiler version
configure:16694: gcc --version >&5
gcc (Ubuntu/Linaro 4.6.3-1ubuntu5) 4.6.3
```

**Comment by docbacardi** [comment]
The environment variable `CC` is defined as `gcc` in the travis environment. This overrides the compiler detection based on the host option. Fixed in 3b1667322f806eb13979d06215d9aca6d9bd1e29.
